src/modules/condition/failed.rs

```diff
@@ -80,23 +80,27 @@ impl TranslateModule for Failed {
             if self.is_question_mark {
                 // if the failed expression is in the main block we need to clear the return value
                 let clear_return = if !self.is_main {
-                    let (name, id, variant) = meta.fun_name.clone().expect("Function name not set");
-                    format!("__AF_{name}{id}_v{variant}=''")
+                    let fun_name = meta.fun_name.as_ref().expect("Function name not set");
```

Comment:
Bundled `(name, id, variant)` into a new struct `FunctionName`, because we were duplicating mangled name generation in three places.